examples/webhook-signing/test/main.ts

```diff
@@ -112,7 +112,11 @@ const sendTestRequest = async (url: string): Promise<string> => {
     process.exit(0);
   } catch (e) {
     console.error(e);
+    throw e;
   } finally {
     server.kill();
   }
-})().catch(console.log);
+})().catch((e) => {
```

Comment:
Was a failing test not failing the CI?

Reply:
Before, if the server hadn't started (e.g., if main.ts wasn't executable or `deno` wasn't installed), then the test would still pass, for example in https://github.com/stripe/stripe-node/actions/runs/4696233056/jobs/8326148593 the test exits 1 with:
```
sh: 1: 
deno: not found
```
but CI does not fail.